test/fixtures/output/local/services/kafka/start-help-onprem.golden

```diff
@@ -1,4 +1,4 @@
-Start Apache Kafka®.
+Start Apache Kafka®. Consider using `confluent local kafka` instead for a faster experience. In the next major confluent CLI version, this command will be removed and replaced by ongoing support for `confluent local kafka`.
```

Comment:
if i'm a customer, i would be curious to know when the next major will come out, but that might make a description too long...

Reply:
@brianstrauch Looks good, but I think I'll only add `start` to the first one since the second sentence implies support for the whole `local kafka` set of commands.